### ISSUE TITLE
shabang and syncstate.py

### ISSUE THREAD

**Issue by Germano0** [owncloud/client#4436]
Hi, I am one of the two Fedora package maintainers for Owncloud client.
Today during [Fedora rpmlint process][1] for Owncloud 2.1 on Fedora alpha release, I got some errors like:
```bash
owncloud-client-nautilus.x86_64: E: non-executable-script /usr/share/nautilus-python/extensions/syncstate.py 644 /usr/bin/python3
```
Then I think: If
* syncstate.py is an executable file I will have to follow [this guide][2] to fix the problem;
* syncstate.py is like a library, probably the shabang should be removed.

What do you think about? Thank you very much

[1]: https://taskotron.fedoraproject.org/artifacts/all/7c4c42c2-cb66-11e5-8d1c-525400120b80/task_output/owncloud-client-2.1.0-2.fc24.log
[2]: https://fedoraproject.org/wiki/Packaging_tricks#Exec_permission

**Comment by dragotin** [comment]
Thanks for your input! 

The syncstate.py is a plugin for nautilus, the GNOME file manager. I have no idea if that requires the shebang or not.

**Comment by guruz** [comment]
I'd say no if it is loaded as a plugin and not from the shell?

@ckamm

**Comment by ogoffart** [comment]
I guess you can just remove the exec permission on that file

**Comment by dragotin** [comment]
The shebang can be removed.

**Comment by Germano0** [comment]
Will the commit be included in final 2.1.1 release?
